cwms_radar_api/src/main/java/cwms/radar/data/dao/LocationGroupDao.java

```diff
@@ -26,13 +27,15 @@
 import org.jooq.SelectOrderByStep;
 import org.jooq.SelectSeekStep1;
 import org.jooq.TableField;
-
+import org.jooq.impl.DSL;
 import usace.cwms.db.jooq.codegen.tables.AV_LOC;
 import usace.cwms.db.jooq.codegen.tables.AV_LOC_CAT_GRP;
 import usace.cwms.db.jooq.codegen.tables.AV_LOC_GRP_ASSGN;
 
 public class LocationGroupDao extends JooqDao<LocationGroup> {
 
+    public static final String CWMS = "CWMS";
```

Comment:
perhaps we should take the time to put these in the Enum section for constants. Not now though, just food for thought, it's already duplicated within this PR though.